pdf-converter/app/pdf_processor.py

```
@@ -18,7 +18,7 @@ def encode_image(self, image_path: str) -> str:
         with open(image_path, "rb") as image_file:
             return base64.b64encode(image_file.read()).decode("utf-8")
 
-    def process_pdf(self, uploaded_file, progress_callback=None, content_callback=None) -> pd.DataFrame:
+    def process_pdf(self, uploaded_file, progress_callback=None, content_callback=None, prompt_text: str = None) -> pd.DataFrame:
```

Pylint (3.3.7)

[refactor] Too many local variables (18/15)

(R0914)